package/nativeComponents/layouts/NativeList.js

```diff
@@ -1,8 +1,289 @@
+        break;
+
+      case "MONGOLIAN":
+        styleClasses.push(UtilityClasses.LIST_STYLE.MONGOLIAN);
+        break;
+
+      case "MYANMAR":
+        styleClasses.push(UtilityClasses.LIST_STYLE.MYANMAR);
+        break;
+
+      case "ORIYA":
+        styleClasses.push(UtilityClasses.LIST_STYLE.ORIYA);
+        break;
+
+      case "OUTSIDE":
+        styleClasses.push(UtilityClasses.LIST_STYLE.OUTSIDE);
+        break;
+
+      case "PERSIAN":
+        styleClasses.push(UtilityClasses.LIST_STYLE.PERSIAN);
+        break;
+
+      case "SIMP_CHINESE_FORMAL":
+        styleClasses.push(UtilityClasses.LIST_STYLE.SIMP_CHINESE_FORMAL);
+        break;
+
+      case "SIMP_CHINESE_INFORMAL":
+        styleClasses.push(UtilityClasses.LIST_STYLE.SIMP_CHINESE_INFORMAL);
+        break;
+
+      case "TELUGU":
+        styleClasses.push(UtilityClasses.LIST_STYLE.TELUGU);
+        break;
+
+      case "THAI":
+        styleClasses.push(UtilityClasses.LIST_STYLE.THAI);
+        break;
+
+      case "TIBETAN":
+        styleClasses.push(UtilityClasses.LIST_STYLE.TIBETAN);
+        break;
+
+      case "TRAD_CHINESE_FORMAL":
+        styleClasses.push(UtilityClasses.LIST_STYLE.TRAD_CHINESE_FORMAL);
+        break;
+
+      case "TRAD_CHINESE_INFORMAL":
+        styleClasses.push(UtilityClasses.LIST_STYLE.TRAD_CHINESE_INFORMAL);
+        break;
+
+      case "UPPER_ALPHA":
+        styleClasses.push(UtilityClasses.LIST_STYLE.UPPER_ALPHA);
+        break;
+
+      case "UPPER_ARMENIAN":
+        styleClasses.push(UtilityClasses.LIST_STYLE.UPPER_ARMENIAN);
+        break;
+
+      case "UPPER_LATIN":
+        styleClasses.push(UtilityClasses.LIST_STYLE.UPPER_LATIN);
+        break;
+
+      case "UPPER_ROMAN":
+        styleClasses.push(UtilityClasses.LIST_STYLE.UPPER_ROMAN);
+        break;
+
+      case "URDU":
+        styleClasses.push(UtilityClasses.LIST_STYLE.URDU);
+        break;
+
+      case "INHERIT":
+        styleClasses.push(UtilityClasses.LIST_STYLE.INHERIT);
+        break;
+
+      case "INITIAL":
+        styleClasses.push(UtilityClasses.LIST_STYLE.INITIAL);
+        break;
+
+      case "REVERT":
+        styleClasses.push(UtilityClasses.LIST_STYLE.REVERT);
+        break;
+
+      case "REVERT_LAYER":
+        styleClasses.push(UtilityClasses.LIST_STYLE.REVERT_LAYER);
+        break;
+
+      case "UNSET":
+        styleClasses.push(UtilityClasses.LIST_STYLE.UNSET);
+        break;
+          
+      default:
+        styleClasses.push(UtilityClasses.LIST_STYLE.AUTO);
+    }
+
+    return [...styleClasses, ...(props?.styleClasses || [])];
+  };
+  const listItemStyleClasses = (childProps) => {
+    let styleClasses = [];
+
+    if (props.listType) {
+      styleClasses.push(UtilityClasses.DISPLAY.LIST_ITEM, UtilityClasses.PADDING.P0);
+    }
+    return [...styleClasses, ...(childProps?.styleClasses || [])];
+  };
+
+  return <CoreBox styleClasses={[UtilityClasses.PADDING.PL2]}><SCList
```

Comment:
Please try to remove this unnecessary CoreBox it will cause our default list behavior abnormal.

Reply:
without the use of corebox the padding is not implemented. i will try if there is a way to do this work.